app/ui/src/app/customizations/tech-extensions/import/tech-extension-import.component.ts

```diff
@@ -70,13 +71,14 @@ export class TechExtensionImportComponent implements OnInit {
   }
 
   ngOnInit() {
-    this.extensionId = this.route.snapshot.paramMap.get('id');
-    this.extensionName = this.route.snapshot.paramMap.get('name');
+    this.id = this.route.snapshot.paramMap.get('id');
+    this.extensionId = this.route.snapshot.paramMap.get('extensionId');
+    this.extensionName = this.route.snapshot.queryParamMap.get('name');
     if (!this.extensionName) {
       // safety net
       this.extensionId = undefined;
```

Comment:
Shouldn't be "safety net" also changed to `this.id = undefined;`? Otherwise 👍

Reply:
I do not know. The thing is that I do not understand the logic behind that LOC nor its purpose. I actually do not understand anything from this file and am just following my gut since Angular apps do not operate this way, but I guess you've got a point, since we will want to keep the same previous logic prior to execute `this.extensionStore.getUploadUrl()`. I'll push a new commit...